pkg/appmanager/resourceConfig.go

```diff
@@ -719,21 +749,18 @@ func copyConfigMap(virtualName, ns string, cfg *ResourceConfig, cfgMap *ConfigMa
 // Create a ResourceConfig based on an Ingress resource config
 func createRSConfigFromIngress(
 	ing *v1beta1.Ingress,
+	resources Resources,
```

Comment:
This should be a *Resources since it contains locks and shouldn't be copied.

Reply:
Good catch. I need to start keeping this in mind more now.